tests/test_autotuner.py

```diff
@@ -96,7 +96,7 @@ def test_mode_destructive():
     grid = Grid(shape=(96, 96, 96))
     f = TimeFunction(name='f', grid=grid, time_order=0)
 
-    op = Operator(Eq(f, f + 1.), dle=('advanced', {'openmp': False}))
+    op = Operator(Eq(f.forward, f + 1.), dle=('advanced', {'openmp': False}))
```

Comment:
why?

Reply:
I think it was just a leftover. I have reinstated the origin version